docs/openapi.yaml

```diff
@@ -681,6 +681,52 @@ paths:
       security:
         - basic_auth: []
 
+  # In theory not needed if `required: false` work in the next API call, but it doesn't
```

Comment:
I slightly misunderstood this verbiage, suggestion:

```diff
This /profiling/ endpoint is separate from the next API call (/profiling/{handler})
because `required: false` behaved as expected.  Since it is not,
/profiling/ will route to a separate method that just calls the /profiling/{handler}
endpoint with the empty string
```

Or something.  Maybe that is a little too verbose, but something explaining "this section really only exists because of a `required: false` bug in the next section" sort of sentiment.  If instead of `next API call` it said `/profiling/{handler}` I probably would have understood that more clearly :slightly_smiling_face:

Reply:
Yep, great suggestion! Added in 07c1304b801d570b1c9de6e010d27576abe9dbf3

Reply:
I think you missed a negative:

```
  # This /profiling/ endpoint is separate from the /profiling/{handler} because `required: false`
  # behaved as expected. Since it is not, /profiling/ will route to a separate method that just
  # calls the /profiling/{handler} endpoint with the empty string
```

I think it should be

```
# ... because `required:false`
# did *NOT* behave as expected....
```

Second line, ~~behaved as expected~~ => `did *NOT* behave as expected`.

Since we have one more small fixup here, the third line, lets add a final punctuation (add a `.` after `endpoint with the empty string`).

Reply:
Yep, added here: 4975dea9c2cec01bb465e429c7f09b3844c4db1e

Reply:
I realize now I was the one who made that typo :sweat_smile:  Thanks for fixing up the docs, looks great!